docs/versioned_docs/version-2.3.0/api/test-api.md

```diff
@@ -3,6 +3,10 @@ id: test-api
 title: Testing
 ---
 
+:::info
+If you want to use `fireGestureHandler` and `getByGestureTestId`, you need to import them from `react-native-gesture-handler/jest-utils` package.
+:::
```

Comment:
Change the newest md as well, please.

Reply:
For now I have moved it to the newest one in https://github.com/software-mansion/react-native-gesture-handler/pull/1917/commits/7326e847e7d987e550d823273f3d615b2ae89535. It's kinda breaking change and I'm not sure whether we should release it as a patch.